spec/lib/spotlight/dor/indexer_spec.rb

```diff
@@ -120,6 +120,43 @@
     end
   end
 
+  describe '#add_document_subtype' do
```

Comment:
The last time I looked at this code, I was thinking it would be good to organize the specs into the same concerns?  Or alphabetically or something?  This file is getting kind of long.